tests/conftest.py

```diff
@@ -1,9 +1,13 @@
+import os
 from dataclasses import dataclass
 from pathlib import Path
 
 import pytest
 
-import hatch.config
+
+# set a testing secret
+os.environ["BACKEND_TOKEN"] = "secret"
+import hatch.config  # noqa
```

Comment:
Can you suffix what error the `noqa` is suppressing here, otherwise we're ignoring all warnings/errors the line! (eg `# noqa: E123`